sdk/src/transaction_context.rs

```diff
@@ -706,7 +916,22 @@ impl From<TransactionContext> for ExecutionRecord {
                 .collect(),
             instruction_trace: context.instruction_trace,
             return_data: context.return_data,
+            changed_account_count,
+            total_size_of_all_accounts,
+            total_size_of_touched_accounts,
             total_resize_delta: RefCell::into_inner(context.total_resize_delta),
         }
     }
 }
+
+fn is_zeroed(buf: &[u8]) -> bool {
+    const ZEROS_LEN: usize = 1024;
+    static ZEROS: [u8; ZEROS_LEN] = [0; ZEROS_LEN];
```

Comment:
nit: you can now use `const` here. :) this carried code is surely dated from pre const rusts. ;)